src/app/(profile)/profiles/[position]/page.tsx

```diff
@@ -0,0 +1,35 @@
+﻿import ProfileList from '@/app/(profile)/(components)/ProfileList/ProfileList'
+import {
+  filterByAvailability,
+  filterByFullName,
+  filterByLocation,
+  filterBySeniority,
+  filterByTechnology,
+} from '@/app/(profile)/(components)/ProfileList/filters'
+import { type JobOfferFiltersEnum } from '@/app/(profile)/types'
+import { getPublishedProfilesPayloadByPosition } from '@/backend/profile/profile.service'
+import Loader from '@/components/Loader/Loader'
+import { Suspense } from 'react'
+
+export default async function PositionPage({
```

Comment:
**nit:** We have to be consistent in terms of domain - let's make it `SpecializationPage`, and `/profiles/[specialization]/page.tsx`. I know we have `position` in DB but we will have to change it for sure